<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndFilters</class>
 <widget class="QDialog" name="wndFilters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>298</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>372</width>
    <height>298</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add filter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lbName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbComments">
     <property name="text">
      <string>Comment regex:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edComments"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="gbFiles">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QTreeWidget" name="twFilesInclude">
        <property name="indentation">
         <number>0</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTreeWidget" name="twFilesExclude">
        <property name="indentation">
         <number>0</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbFilesInclude">
        <property name="text">
         <string>Include:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbFilesExclude">
        <property name="text">
         <string>Exclude:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="layHorizontal_3">
        <item>
         <spacer name="spHorizontal_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnFilesExcludeAdd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFilesExcludeRemove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="layHorizontal_2">
        <item>
         <spacer name="spHorizontal_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnFilesIncludeAdd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFilesIncludeRemove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="layHorizontal">
     <item>
      <spacer name="spHorizontal">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edName</tabstop>
  <tabstop>edComments</tabstop>
  <tabstop>twFilesInclude</tabstop>
  <tabstop>twFilesExclude</tabstop>
  <tabstop>btnFilesIncludeAdd</tabstop>
  <tabstop>btnFilesIncludeRemove</tabstop>
  <tabstop>btnFilesExcludeAdd</tabstop>
  <tabstop>btnFilesExcludeRemove</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>wndFilters</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
